SMP/gnutls/gnutls.h

```diff
@@ -39,7 +39,9 @@
 /* Get ssize_t. */
 #ifdef _MSC_VER
 #  include <BaseTsd.h>
+#ifndef ssize_t
```

Comment:
Should be indented i.e. `#  ifndef` otherwise looks fine

Reply:
done.